dali/test/python/test_pipeline.py

```diff
@@ -738,23 +752,17 @@ def test_lazy_init_empty_data_path():
     batch_size = 128
 
     nonlazy_pipe = LazyPipeline(batch_size, empty_db_folder, lazy_type=False)
-    try:
+    with assert_raises(RuntimeError):
         nonlazy_pipe.build()
-        assert(False)
-    except RuntimeError:
-        assert(True)
 
     lazy_pipe = LazyPipeline(batch_size, empty_db_folder, lazy_type=True)
-    try:
-        lazy_pipe.build()
-        assert(True)
-    except BaseException:
-        assert(False)
+    lazy_pipe.build()
```

Comment:
missing `assert_raises` here?

Reply:
The example above had different order of asserts, I think this one was meant to simply pass (as opposed to the one above that should raise).